robot_cameraman/server.py

```diff
@@ -125,6 +125,35 @@ def angle():
     cameraman_mode_manager.angle(pan_angle=pan_angle, tilt_angle=tilt_angle)
     return '', 204
 
+@app.route('/api/relcoords')
```

Comment:
My thought was that for a quick and dirty initial implementation I would have my tag (phone or watch) call this endpoint directly. I would initialize it at the base and have that be the origin for the coordinate system. It would subtract that origin GPS coordinate from the current GPS coordinate and call /api/relcoords with the values.